pkg/machine/wsl/util_windows.go

```diff
@@ -235,14 +202,6 @@ func reboot() error {
 		}
 	}
 
-	if err := addRunOnceRegistryEntry(command); err != nil {
```

Comment:
This is a critical part of the functionality

Reply:
I didn't delete it, I moved it below.

https://github.com/containers/podman/pull/22922/files#diff-246f5cb53b6a855b52bd19acbaf15f3dfd710da91b35828a9a30bbc295f03f4fR215

Reply:
ah sorry missed that